charts/schema-registry/values.yaml

```diff
@@ -29,6 +29,26 @@ service:
   type: LoadBalancer
   port: 9092
 
+initContainer: {}
+  # name: ""
+  # image:
+  #   repository:
+  #   tag:
+  #   pullPolicy: IfNotPresent
+  # command: []
+  # resources: {}
+  # volume:
+  #   name: ""
+  #   mountPath: ""
+
+container: {}
+  ## just provide the command to be executed within the entrypoint
+  ## e.g. "cp /additional_libs/* /opt/someprogram/libs & ls /opt/someprogram/libs & sleep 10000"
```

Comment:
I would put a more realistic command example to really show the destination where one would have to copy the libraries for the schema registry image.    

( And drop the sleep 10000 example )

Reply:
done